deploy/kubernetes/helm/sloth/tests/helm_chart_test.go

```diff
@@ -365,8 +365,13 @@ func TestChartConfigMap(t *testing.T) {
 				require.NoError(err)
 				expTplS := strings.TrimSpace(string(expTpl))
 
-				assert.Equal(expTplS, gotTpl)
+				assert.Equal(expTplS, NormalizeVersion(gotTpl))
 			}
 		})
 	}
 }
+
+func NormalizeVersion(tpl string) string {
```

Comment:
Its a private method not used outside the package. It should be `normalizeVersion`.

Reply:
Thanks, haven't really written in go, so don't know the nomenclature

Reply:
There is always a first time :smile:  Thanks!